### PR TITLE
Use script to run CAPI tests

### PR DESCRIPTION
**Pull request by alexander-demicev**

### CODE REVIEW

File: ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml [hunk 1]
```diff
@@ -45,7 +45,7 @@ resources:
 test_binary_build_commands: make test
```
Comment (lobziik) [inline ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml:45]: should this also be changed?

File: ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ resources:
...
 tests:
 - as: unit
-  commands: make test
```
Comment (lobziik) [inline ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml:48]: Why do we want to refuse using make here?

Comment (alexander-demichev) [reply ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml:48]: For this case, it's easier than patching upstream one

**Review by elmiko — COMMENTED** [review]
/lgtm

**Pull request merged**